src/Lachain.Core/RPC/HTTP/Web3/BlockchainServiceWeb3.cs

```diff
@@ -66,13 +66,19 @@ public class BlockchainServiceWeb3 : JsonRpcService
             var block = _blockManager.GetByHeight((ulong)blockNumber);
             if (block == null)
                 return null;
-            var txs = block!.TransactionHashes
+
+            ulong gasUsed = 0;
+            var txArray = new JArray();
+            if(block.TransactionHashes.Count>0)
```

Comment:
Should we put an extra space before and after 0 ?

Reply:
```suggestion
            if (block.TransactionHashes.Count > 0)
```
You can just accept the suggestion to fix